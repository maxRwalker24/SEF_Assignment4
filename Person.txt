56s_d%&fAB Max Walker 32|Highland Street|Melbourne|Victoria|Australia 15-11-1990
78AA@@BBCC Jane Smith 45|Main Avenue|Sydney|Victoria|Australia 01-01-1985
98ZZ##YYXX John Doe 12|Park Road|Brisbane|Victoria|Australia 31-12-2000
35s_d%&fAB Paul Vennat 17|Landscape Drive|Melbourne|Victoria|Australia 10-09-1999
36s_d#@fAJ Ajay Peeris 26|YeahNah Road|Melbourne|Victoria|Australia 26-02-2014
26s_d#@fAJ Evan Nombour 22|Salmon Road|Melbourne|Victoria|Australia 21-04-2000
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2007
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2007
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2007
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2007
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
56s_d%&fAB Anthony Duiker 29|Landscape Drive|Melbourne|Victoria|Australia 10-09-2001
