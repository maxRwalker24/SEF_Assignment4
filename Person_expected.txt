56s_d%&fAB Max Walker 32|Highland Street|Melbourne|Victoria|Australia 15-11-1990
78AA@@BBCC Jane Smith 45|Main Avenue|Sydney|Victoria|Australia 01-01-1985
98ZZ##YYXX John Doe 12|Park Road|Brisbane|Victoria|Australia 31-12-2000